src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
```

Comment:
You can't change function's arguments 
```suggestion
  const oldSeparator = fromFormat[3];
  const newSeparator = toFormat[3];
  const startFormat = fromFormat.slice(0, 3)
  const EndFormat = toFormat.slice(0, 3)
```

Reply:
yes Ihor is right, you really can`t change function's arguments